docs/getting-started-scalardb-analytics-postgresql.md

```diff
@@ -0,0 +1,508 @@
+|                                            +------------------------------------+                                                         |
+|                                      +---> | ScalarDB Analytics with PostgreSQL | ---+         +-----------------------------+            |
+|                                      |     +------------------------------------+    |   +---> |  MySQL ("customer" schema)  | <---+      |
+|                                      |                                               |   |     +-----------------------------+     |      |
+|  +-------------+      +---------+    |     +------------------------------------+    |   |                                         |      |
+|  | OLAP client | ---> | Service | ---+---> | ScalarDB Analytics with PostgreSQL | ---+---+                                         +---+  |
+|  +-------------+      +---------+    |     +------------------------------------+    |   |                                         |   |  |
+|                                      |                                               |   |     +-----------------------------+     |   |  |
+|                                      |     +------------------------------------+    |   +---> | PostgreSQL ("order" schema) | <---+   |  |
+|                                      +---> | ScalarDB Analytics with PostgreSQL | ---+         +-----------------------------+         |  |
+|                                            +------------------------------------+                                                      |  |
+|                                                                                                                                        |  |
+|  +-------------+                                                                                                                       |  |
+|  | OLTP client | ---(Load sample data with a test OLTP workload)-----------------------------------------------------------------------+  |
+|  +-------------+                                                                                                                          |
+|                                                                                                                                           |
++-------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+## Step 1. Start a Kubernetes cluster
+
+First, you need to prepare a Kubernetes cluster. If you're using a **minikube** environment, please refer to the [Getting Started with Scalar Helm Charts](./getting-started-scalar-helm-charts.md). If you have already started a Kubernetes cluster, you can skip this step.
+
+## Step 2. Start MySQL and PostgreSQL pods
+
+ScalarDB including ScalarDB Analytics with PostgreSQL can use several types of database systems as a backend database. In this guide, we will use MySQL and PostgreSQL.
```

Comment:
Sorry, I missed this mention of `we`.

```suggestion
ScalarDB including ScalarDB Analytics with PostgreSQL can use several types of database systems as a backend database. In this guide, you will use MySQL and PostgreSQL.
```

Reply:
No problem! Thank you for pointing it out! 🙇‍♀️